src/main/docker/Dockerfile

```
@@ -27,11 +27,11 @@ RUN if [ "${cantaloupe.version}" = "dev" ] ; then \
       fi && \
       \
       # We can also supply a patchfile as a defense against a broken upstream
-      if [ -n "${cantaloupe.patchfile}" ] ; then \
-        if [ -f "/build/cantaloupe-patches/${cantaloupe.patchfile}" ] ; then \
-          git apply "/build/cantaloupe-patches/${cantaloupe.patchfile}" ; \
+      if ${cantaloupe.apply.patchfiles} ; then \
+        if [ $(ls -la /build/cantaloupe-patches/**.patch | wc -l) -gt 0 ] ; then \
+          git apply $(ls /build/cantaloupe-patches/**.patch | xargs) ; \
```

If this was intended to match files in subdirectories, I think it needs a change:

```suggestion
        if [ $(ls -la /build/cantaloupe-patches/**/*.patch | wc -l) -gt 0 ] ; then \
          git apply $(ls /build/cantaloupe-patches/**/*.patch | xargs) ; \
```

---

So the .patch files are actually copied to `/build/cantaloupe-patches` The output of git apply would be:

```git apply /build/cantaloupe-patches/issue-555-v5.0.5.patch /build/cantaloupe-patches/issue-574-v5.0.5.patch /build/cantaloupe-patches/issue-581-v5.0.5.patch```

There isn't a subdirectory under it. I initially tried `/build/cantaloupe-patches/*.patch`, it worked in bash in my terminal, but for some reason it returns an error under the Dockerfile RUN layer. Changing it to `/build/cantaloupe-patches/**.patch` seems to work though.

---

If you want to test, you can run this after checking out the branch:

```mvn clean verify -Dkakadu.git.repo=scm:git:git@github.com:uclalibrary/kakadu.git -Dkakadu.version=v7_A_7-01903E -Dcantaloupe.version=dev -Dcantaloupe.commit.ref=096ffed0056d0a86414307efce499609b07720eb -Dcantaloupe.apply.patchfiles=true -Ddocker.noCache```

---

Huh, okay then. Sorry for making that assumption.